mobile/src/toptabbar/people-top-tab-bar-model.tsx

```diff
@@ -0,0 +1,30 @@
+import { TabNavigatorConfig, NavigationRouteConfigMap } from 'react-navigation';
+import React from 'react';
+import { PeopleDepartment } from '../people/people-department';
+import { PeopleRoom } from '../people/people-room';
+import { PeopleCompany } from '../people/people-company';
+import { EmployeesList } from '../people/employees-list';
+import { TabNavigationOptionsFactory } from '../tabbar/tab-navigation-options-factory';
+
+const navOptionsFactory = new TabNavigationOptionsFactory();
+
+const peopleTopTabBarModel: NavigationRouteConfigMap = {
```

Comment:
Get rid of this folder, move it under people. It's only used in people scene.

Reply:
Or, rather, you can leave this and styles file in this folder, but get rid of people-top-tab-bar